db/migrate/20190102182940_remove_join_table.rb

```diff
@@ -0,0 +1,15 @@
+class RemoveJoinTable < ActiveRecord::Migration[5.1]
+  def up
+    drop_join_table :portfolios, :portfolio_items
+    add_column :portfolio_items, :portfolio_id, :bigint
```

Comment:
@syncrou As we discussed, would be good to preserve the relationship between an item and one of the portfolios here.  I do not think we need to duplicate items to get back to the same view the previous HABTM relationship would have shown, but just connecting existing items to one of the portfolios.  Since this is all test environments now it will not matter and it keeps from having orphaned portfolio_items left in the db.

Reply:
I added in changes to migrate data on the up and down action.